notebooks/benchmarks_sandbox.ipynb

```diff
@@ -9,7 +9,7 @@
     "\n",
```

Comment:
<p>Lots of convergence warnings here; I wonder if its worth increasing the number of iterations just to see.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Yes! This is why I used the StandardScaler as attempt at a solution. I'll try more iterations